felix/bpf/tc/compiler.go

```diff
@@ -101,7 +101,7 @@ func ProgFilename(epType EndpointType, toOrFrom ToOrFromEp, epToHostDrop, fib, d
 	case EpTypeHost:
 		epTypeShort = "hep"
 	case EpTypeTunnel:
-		if bpf.IPIPDeviceIsL3() {
+		if features != nil && features.IPIPDeviceIsL3 {
```

Comment:
This cannot be `nil` because you are taking a pointer of a struct when calling this function from `func (ap AttachPoint) FileName()`